README.md

```diff
@@ -93,19 +93,20 @@ Note that states and issue transitions are based on your Jira project workflow c
 
 #### Step 2: Configure webhooks in Jira
 
-If you want to [send notifications from Jira to Mattermost](#11-send-notifications-from-Jira-to-Mattermost), link a Jira project to a Mattermost channel via webhooks.
+As of Jira 2.1, the plugin gets sent a firehose of events from the server via the webhook below.  Previously configured webhooks that point to specific channels are still supported and will continue to work. 
```

Comment:
1/5
> As of Jira 2.1, the plugin ~~gets sent~~ **will process** a firehose of events

0/5 maybe explain what firehose means, or possibly use different terminology. Or explain that the configuration is now on MM side instead of configuring separate webhooks in Jira.

There is some more explanation below, but the wording still seems a bit foreign to me.

Is possible to link to the `How do I manually Configure webhooks notifications to be sent to a Mattermost Channel?` section of the readme from here?